datasets/feverous.yaml

```diff
@@ -0,0 +1,28 @@
+Name: 'FEVEROUS: Fact Extraction and VERification Over Unstructured and Structured information'
+Description: FEVEROUS (Fact Extraction and VERification Over Unstructured and Structured information) is a
+  fact verification dataset which consists of 87,026 verified claims. Each claim is annotated with evidence in
+  the form of sentences and/or cells from tables in Wikipedia, as well as a label indicating whether this evidence
+  supports, refutes, or does not provide enough information to reach a verdict.
+Documentation: https://fever.ai/resources.html
+Contact: fever-organisers@googlegroups.com
+ManagedBy: '[FEVER team](https://fever.ai)'
+UpdateFrequency: Not updated
+Tags:
+  - natural language processing
+  - machine learning
+  - fact verification
+License: '[CC BY-SA 3.0](https://creativecommons.org/licenses/by-sa/3.0/)'
+Resources:
+  - Description: Project data files in a public bucket
+    ARN: arn:aws:s3:::fever.public/feverous/
```

Comment:
It does not look like this S3 bucket is publicly accessible? You can see an example 'open data' bucket policy at https://gist.github.com/jedsundwall/a5d07ae3971fb172cf923d8de4625536.

Reply:
I have made the individual objects public e.g. https://s3-eu-west-1.amazonaws.com/fever.public/feverous/train.jsonl

As this is an Amazon AWS account, we have been asked not to make S3 buckets themselves publicly available. Is this a deal-breaker for the registry? 

If it helps, we can have an internal chat on Chime/Slack - I'm chrchrs@